script.js

```diff
@@ -1,4 +1,296 @@
+//Initializing Variables
+var mode = "HOME_TURF";
+var player1Array = [];
+var player2Array = [];
+var randomArray = [];
+var myOutputValue = "";
+var randomNumberOnly = 0;
+var randomNumber1 = 0;
+var randomNumber2 = 0;
+var P1FinalNumber = 0;
+var P2FinalNumber = 0;
+var P1_Record = 0;
+var P2_Record = 0;
+var condition = "";
+var LCN_Final_Number_1 = 0;
+var LCN_Final_Number_2 = 0;
+var LCN_Final = 0;
+var P1_max = 0;
+var P2_max = 0;
+var Final_P1_VND = 0;
+var Final_P2_VND = 0;
+
+//Generate Random Number from 1 to 6
+var rollDice = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+//Check Condition for which player wins + Generating Record (Normal)
+var checkCondition = function (x, y) {
+  if (x > y) {
+    condition = "Player 1 Wins!";
+    P1_Record += 1;
+  } else if (y > x) {
+    condition = "Player 2 Wins!";
+    P2_Record += 1;
+  } else {
+    P1_Record = P1_Record;
+    P2_Record = P2_Record;
```

Comment:
Hi Wira, for this part, there is no need resassign because, both P1_Record and P2_Record will remain the same without us having to assign like this